frontend/packages/kubevirt-plugin/integration-tests/tests/dialogs/nodeSelectorDialog.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-await-in-loop */
+import * as view from '../../views/dialogs/editNodeSelectorView';
+import { fillInput } from '@console/shared/src/test-utils/utils';
+import { click } from '@console/dev-console/integration-tests/utilities/elementInteractions';
+import { MatchLabels } from '@console/internal/module/k8s';
+
+export class NodeSelectorDialog {
+  private extractID(id: string) {
+    return id.split('-')[1];
+  }
+
+  /**
+   * Adds new row if needed.
+   * Returns string with index of next empty row.
+   */
+  async addRow(): Promise<string> {
```

Comment:
Other place like `Tolerations` and `affinity` may use `addRow` as well, then we can move it to `utils` or `console-shared`.

Reply:
Better place would probably be some scheduling.dialog.view.ts file, utils and console-shared are for more general purpose utility methods

Reply:
okay, if you decide not to do it in this one, then it looks good to me.